### ISSUE TITLE
App closes itself after splash screen

### ISSUE THREAD

**Issue by MTVirux** [wanjizheng/iBarter_WPF#3]
Hello there!
First time trying to use the app. I've installed and repaired the runtime multiple times between attempts.

Order of events happening:

1. I run the app as admin
2. Splashscreen pops up
3. Loads a bit
4. Focuses my game
5. Resizes the window (I play in windowed mode)
6. Splash screen disappears 
7. Resizes my game back to what it was
8. Closes itself after a while

I'm running Windows 11 Pro 10.0.22631 Build 22631
Also tried changing the game to fullscreen and windowed fullscreen

EDIT: Beta 2 starts but any interaction with the app causes it to crash

**Comment by legacy3** [comment]
I second this issue. Will try to look into it myself, but it's def broken for me aswell.

**Comment by legacy3** [comment]
I looked into this issue, and it's not really possible to fix. The core of this program is a DLL (PureDM.dll) loaded via NuGet, and the WPF app connects to it by passing an API key. I unpacked the DLL (packed with UPX) and used IDA to analyze it. It appears that PureDM is a bypass DLL for Xigncode. Be warned that running this program might be risky for your game account. Despite successfully unpacking the UPX-packed DLL and examining it, I found no definitive way to determine what it does to your PC. The program can spawn random processes, performs some unusual actions with cmd.exe, and connects you to a random server. This is not a local operation.

TL;DR: I'm not suggesting this is malicious, but be cautious of what you run. Also, this software isn't entirely open source; only the non-critical part is. Thus, fixing this issue is not possible.

**Comment by wanjizheng** [comment]
PureDM.dll is the package I use to:
1. bind the game window.
2. take screenshots from the game window.
3. convert the screen coordinates to game coordinates.
4. ORC and image recognition (Emgu CV).

Yes, the application will be blocked by your anti-virus software and close automatically unless you white list this file.

**Comment by Kamiikaze** [comment]
Even whitelisting file + folder, just closes the app after splash screen

**Comment by wanjizheng** [comment]
> j

Ok, just noticed today that the PureDM.dll has some compatibility issues with the latest Windows 11 updates. I'll update the version shortly to fix this problem.

**Comment by wanjizheng** [comment]
Beta 4.0 was released, and the compatibility issue was fixed.
